package.json

```diff
@@ -50,7 +50,7 @@
     "@babel/preset-env": "^7.5.5",
     "@babel/preset-flow": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
-    "@popperjs/core": "^2.0.0",
+    "popper.js": "^1.14.4",
```

Comment:
We should be using consistent versions for both this lib and `react-popper`

Reply:
Hopefully someone is revamping that react-popper library for 2.0 popper support AND one day it will be super easy to use... 🤞 🤞

Reply:
@jasperfurniss https://github.com/popperjs/react-popper/issues/332